File: app/src/main/java/novajox/io/healthfit/data/Record.java
package novajox.io.healthfit.data;

import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;

/**
 * Created by Jocelyn on 16/07/2017.
 */
@Table(database = AppDatabase.class)
public class Record {

    @PrimaryKey(autoincrement = true)
    Integer id;
    public Long date;
    public Float totalMass;
    public Float fatMass;
    public Float muscleMass;
    public Float waterMass;
}

File: app/src/main/java/novajox/io/healthfit/data/Record.kt
package novajox.io.healthfit.data

import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.PrimaryKey
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.structure.BaseModel

/**
 * Created by Jocelyn on 16/07/2017.
 */
@Table(database = AppDatabase::class)
class Record : BaseModel {

    @PrimaryKey(autoincrement = true)
    var id: Int? = null
    @Column
    var date: Long? = null
    @Column
    var totalMass: Float? = null
    @Column
    var fatMass: Float? = null
    @Column
    var muscleMass: Float? = null
    @Column
    var waterMass: Float? = null

    constructor() {}

}
